ui/widgets/auxcontrols.go

```diff
@@ -126,6 +136,41 @@ func (a *AuxControls) OnShowCastMenu(f func(func())) {
 	}
 }
 
+func (a *AuxControls) showSpeedMenu() {
+	speeds := []float64{0.25, 0.5, 0.75, 1.0, 1.25, 1.5, 1.75, 2.0, 2.25, 2.5, 2.75, 3.0, 3.25, 3.5, 3.75, 4.0}
```

Comment:
I would prefer instead of a huge menu like this if the menu had fewer options (e.g. 0.25, 0.5, 0.75, 1.0, 1.25, 1.5, 1.75, 2, 4) plus a "Custom" option that would launch a PopUp with a slider to set a custom speed outside these options. That is something else I can do in a follow-up if you prefer.